### PR TITLE
add AAI blogpost

### PR DESCRIPTION
**Pull request by bgruening**
ping @erasche

### CODE REVIEW

File: _posts/2019-06-13-elixir-aai-done.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ - denbi
+---
+
+As of Galaxy version 19.05 released last month, Galaxy can easily be configured to use Elixir-AAI service for authenticating users! As the first publicly available Galaxy instance, the [European Galaxy Server](https://usegalaxy.eu/) hosted by Elixir Germany is now ready to accept logins from users using their Elixir AAI identity for authentication.
+
+The Elixir AAI integration into Galaxy main source code is one of the deliverables in the Elixir Galaxy Community Implementation Study (reference/link), and was achieved through  collaboration between Elixir Norway, Elixir Germany, Elixir Finland, Elixir Czech Republic and the Galaxy core developers.
+
+The integration is based upon the OpenID Connect functionality in Galaxy, implemented through the Python Social Auth library. Galaxy users with a registered Elixir AAI user profile, simply login by following the “Login with Elixir” button:
+
+![ELIXIR AAI Integration](/assets/media/elixir_aai.png)
```
Comment (erasche) [inline _posts/2019-06-13-elixir-aai-done.md]: missing when I received ping

File: _posts/2019-06-13-elixir-aai-done.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+
+[User documentation](https://galaxyproject.org/authnz/use/oidc/idps/elixir-aai/) at galaxyproject.org official Galaxy documentation site.
```
Comment (erasche) [inline _posts/2019-06-13-elixir-aai-done.md]:
```suggestion
[User](https://galaxyproject.org/authnz/use/oidc/idps/elixir-aai/) and [Admin](https://galaxyproject.org/authnz/config/oidc/idps/elixir-aai/) documentation is available on the Galaxy Hub.
```

File: _posts/2019-06-13-elixir-aai-done.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+ 
+If other Galaxy instances also would like to accept Elixir AAI logins, they need to register with Elixir AAI as a Service Provider and configure their Galaxy instance accordingly. Galaxy admin documentation for the setup is provided here.
+
+[Admin documentation](https://galaxyproject.org/authnz/config/oidc/idps/elixir-aai/) at galaxyproject.org official Galaxy documentation site.
```
Comment (erasche) [inline _posts/2019-06-13-elixir-aai-done.md]:
```suggestion
```

File: _posts/2019-06-13-elixir-aai-done.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+The Elixir AAI integration into Galaxy main source code is one of the deliverables in the Elixir Galaxy Community Implementation Study (reference/link), and was achieved through  collaboration between Elixir Norway, Elixir Germany, Elixir Finland, Elixir Czech Republic and the Galaxy core developers.
+
+The integration is based upon the OpenID Connect functionality in Galaxy, implemented through the Python Social Auth library. Galaxy users with a registered Elixir AAI user profile, simply login by following the “Login with Elixir” button:
+
+![ELIXIR AAI Integration](/assets/media/elixir_aai.png)
+
+
+[User documentation](https://galaxyproject.org/authnz/use/oidc/idps/elixir-aai/) at galaxyproject.org official Galaxy documentation site.
+ 
+If other Galaxy instances also would like to accept Elixir AAI logins, they need to register with Elixir AAI as a Service Provider and configure their Galaxy instance accordingly. Galaxy admin documentation for the setup is provided here.
```
Comment (erasche) [inline _posts/2019-06-13-elixir-aai-done.md]:
```suggestion
If other Galaxy instances also would like to accept Elixir AAI logins, they need to register with Elixir AAI as a Service Provider and configure their Galaxy instance accordingly. Documentation for the admin setup is provided.
```

File: _posts/2019-06-13-elixir-aai-done.md (hunk 1 above)
Comment (bgruening) [reply _posts/2019-06-13-elixir-aai-done.md]: forgot to commit ... damn

**Pull request merged**